PORTAL/jobs/_workers.py

```diff
@@ -104,6 +105,10 @@ def __repr__(self):
     def __eq__(self, other):
         raise NotImplementedError
 
+    @property
+    def name(self):
+        return self._name
```

Comment:
FWIW, I've had to change "name" to "id" so many times on other projects (because we needed "name" for something else and not changing it was confusing) that I usually call this "id" from the start.